targets/targets.json

```diff
@@ -12525,7 +12525,7 @@
             }
         },
         "components_add": [
-            "FLASHIAP"
+            "OSPIF"
```

Comment:
Why FLASHIAP is removed ?

Reply:
Before I add the OSPI drvier, I have executed the test "features-storage-tests-blockdevice-general_block_device" of FLASHIAP, but it faild. 

Have you test FLASHIAP on DISCO_L4R9I before?

You can see the error information below. 

mbedgt: processing target 'DISCO_L4R9I' toolchain 'GCC_ARM' compatible platforms... (note: switch set to --parallel 1)
mbedgt: test case filter (specified with -n option)
	features-storage-tests-blockdevice-general_block_device
	test filtered in 'features-storage-tests-blockdevice-general_block_device'
mbedgt: running 1 test for platform 'DISCO_L4R9I' and toolchain 'GCC_ARM'
mbedgt: mbed-host-test-runner: started
mbedgt: :741::FAIL: Expected -1 Was 0
mbedgt: :741::FAIL: Expected -1 Was 0
mbedgt: :747::FAIL: Expected 0 Was 1
mbedgt: :741::FAIL: Expected -1 Was 0
mbedgt: :747::FAIL: Expected 0 Was 2
mbedgt: :258::FAIL: Expected 0 Was -1
mbedgt: :274::FAIL: Expected 92 Was 23
mbedgt: :274::FAIL: Expected 163 Was 0
mbedgt: :274::FAIL: Expected 105 Was 0
mbedgt: :274::FAIL: Expected 177 Was 0
mbedgt: :274::FAIL: Expected 133 Was 0
mbedgt: :274::FAIL: Expected 106 Was 56

Reply:
> Why FLASHIAP is removed ?

@jeromecoutant

Before I add the OSPI drvier, I have executed the test "features-storage-tests-blockdevice-general_block_device" of FLASHIAP, but it faild. 

Have you test FLASHIAP on DISCO_L4R9I before?

You can see the error information below. 

mbedgt: processing target 'DISCO_L4R9I' toolchain 'GCC_ARM' compatible platforms... (note: switch set to --parallel 1)
mbedgt: test case filter (specified with -n option)
	features-storage-tests-blockdevice-general_block_device
	test filtered in 'features-storage-tests-blockdevice-general_block_device'
mbedgt: running 1 test for platform 'DISCO_L4R9I' and toolchain 'GCC_ARM'
mbedgt: mbed-host-test-runner: started
mbedgt: :741::FAIL: Expected -1 Was 0
mbedgt: :741::FAIL: Expected -1 Was 0
mbedgt: :747::FAIL: Expected 0 Was 1
mbedgt: :741::FAIL: Expected -1 Was 0
mbedgt: :747::FAIL: Expected 0 Was 2
mbedgt: :258::FAIL: Expected 0 Was -1
mbedgt: :274::FAIL: Expected 92 Was 23
mbedgt: :274::FAIL: Expected 163 Was 0
mbedgt: :274::FAIL: Expected 105 Was 0
mbedgt: :274::FAIL: Expected 177 Was 0
mbedgt: :274::FAIL: Expected 133 Was 0
mbedgt: :274::FAIL: Expected 106 Was 56

Reply:
Working in mbed-os master branch:

| target            | platform_name | test suite                                               | result  | elapsed_time (sec) | copy_method |
|-------------------|---------------|----------------------------------------------------------|---------|--------------------|-------------|
| DISCO_L4R9I-ARMC6 | DISCO_L4R9I   | features-storage-tests-blockdevice-buffered_block_device | OK      | 15.64              | default     |
| DISCO_L4R9I-ARMC6 | DISCO_L4R9I   | features-storage-tests-blockdevice-flashsim_block_device | OK      | 15.32              | default     |
| DISCO_L4R9I-ARMC6 | DISCO_L4R9I   | features-storage-tests-blockdevice-general_block_device  | OK      | 26.57              | default     |
| DISCO_L4R9I-ARMC6 | DISCO_L4R9I   | features-storage-tests-blockdevice-heap_block_device     | OK      | 17.56              | default     |
| DISCO_L4R9I-ARMC6 | DISCO_L4R9I   | features-storage-tests-blockdevice-mbr_block_device      | OK      | 17.6               | default     |
| DISCO_L4R9I-ARMC6 | DISCO_L4R9I   | features-storage-tests-blockdevice-util_block_device     | OK      | 16.34              | default     |
| DISCO_L4R9I-ARMC6 | DISCO_L4R9I   | tests-mbed_hal-qspi                                      | OK      | 38.15              | default     |
| DISCO_L4R9I-GCC_ARM | DISCO_L4R9I   | features-storage-tests-blockdevice-buffered_block_device | OK      | 15.58              | default     |
| DISCO_L4R9I-GCC_ARM | DISCO_L4R9I   | features-storage-tests-blockdevice-flashsim_block_device | OK      | 15.26              | default     |
| DISCO_L4R9I-GCC_ARM | DISCO_L4R9I   | features-storage-tests-blockdevice-general_block_device  | OK      | 26.53              | default     |
| DISCO_L4R9I-GCC_ARM | DISCO_L4R9I   | features-storage-tests-blockdevice-heap_block_device     | OK      | 17.63              | default     |
| DISCO_L4R9I-GCC_ARM | DISCO_L4R9I   | features-storage-tests-blockdevice-mbr_block_device      | OK      | 16.44              | default     |
| DISCO_L4R9I-GCC_ARM | DISCO_L4R9I   | features-storage-tests-blockdevice-util_block_device     | OK      | 16.07              | default     |
| DISCO_L4R9I-GCC_ARM | DISCO_L4R9I   | tests-mbed_hal-qspi                                      | OK      | 37.88              | default     |

Reply:
Hi,
I execute the command below, and I got failed result.
    **sudo mbed test -m DISCO_L4R9I -t GCC_ARM -n features-storage-tests-blockdevice-general_block_device**

Could you tell me the command you executed to test many test suite in one time?
Thanks a lot!